crates/uv/src/commands/project/run.rs

```diff
@@ -93,6 +93,7 @@ pub(crate) async fn run(
 ) -> anyhow::Result<ExitStatus> {
     // These cases seem quite complex because (in theory) they should change the "current package".
     // Let's ban them entirely for now.
```

Comment:
Comment feels slightly misplaced but I guess it was here already.

Reply:
Yeah I think that's pretty separate.